script.js

```diff
@@ -0,0 +1,52 @@
+import { taskForm, taskTitle, taskList, btnCreate } from './elements.js';
+
+let tasks = [];
+let taskId = 0;
+
+function openForm() {
+    taskForm.classList.add("show");
+}
+
+function closeForm() {
+    taskForm.classList.remove("show");
+}
+
+function addTask() {
+    const title = taskTitle.value;
+    if (title === "") {
+        alert("Please enter a task title.");
+        return;
+    }
+
+    const newTask = {
+        id: taskId++,
```

Comment:
id should be unique and not like simple 1,2,3. So you can use the time span for generating unique and comparatively better id for each one.